cmd/xk6/main_test.go

```diff
@@ -124,3 +131,38 @@ func TestNormalizeImportPath(t *testing.T) {
 		})
 	}
 }
+
+func TestExpandPath(t *testing.T) {
+	t.Run(". expands to current directory", func(t *testing.T) {
+		got, err := expandPath(".")
+		if got == "." {
+			t.Errorf("did not expand path")
+		}
+		if err != nil {
+			t.Errorf("failed to expand path")
+		}
```

Comment:
Ah, we're not using [`stretchr/testify`](https://github.com/stretchr/testify) in the project yet :disappointed: Otherwise this would be simpler using `require.NoError()` and `assert.NotEqual()`.

It's fine leaving it as is. We can change this everywhere once we add `testify` as a dependency.